Add code for school examples to examples repo #1295

sudip-padhye opened the pull request:
Added PyMC3 code to generate models and later compared with the already available inference data.

sudip-padhye committed:
Added code for centered_eight and non_centered_eight data to examples repo

sudip-padhye committed:
Included PyMC3 code to generate example

OriolAbril commented on centered_eight.ipynb:
```diff
@@ -0,0 +1,233 @@
+{
```
<p>It looks like you have an old version of ArviZ or xarray installed, the default in latest ArviZ is an html representation for InferenceData objects. Can you try updating them? i.e. running <code>pip install -U arviz xarray</code>  (or with conda if you are using conda)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz_example_data/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

OriolAbril replied:
<p>Not sure I would include these last cells, what do you think Ravin? Maybe a cell calling <code>to_netcdf</code>  and storing the result as <code>.nc</code>  file plus adding <code>.nc</code> files to gitignore? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz_example_data/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

OriolAbril commented:
Thanks for the PR! It is looking good! 

We still have not clearly defined how exactly we want the examples to look so we may have to discuss a little to agree on something, we'll try to be quick

sudip-padhye committed:
Updated arviz and xarray underlying versions. Added code to store the model

sudip-padhye commented:
I have uploaded updated files that have following-
1. Underlying arviz and array versions are now updated. You can view the HTML formatted output which was not available in the previous submissions.
2. Also, I have added a code block for saving the model in .nc format using arviz.to_netcdf() API by passing arguments data & filename.

OriolAbril commented:
Thanks!

>     Also, I have added a code block for saving the model in .nc format using arviz.to_netcdf() API by passing arguments data & filename.

Can you add `*.nc` to the `.gitignore`? They should not be included to the repository due to their large size.

canyon289 commented:
> Thanks!
> 
> > ```
> > Also, I have added a code block for saving the model in .nc format using arviz.to_netcdf() API by passing arguments data & filename.
> > ```
> 
> Can you add `*.nc` to the `.gitignore`? They should not be included to the repository due to their large size.

@sudip-padhye in addition to the .nc file should be removed entirely from the git commit tree. Unfortunately there's no easy way to do this other than removing the commit that introduces the file through a rebase, or using a tool like this one https://rtyley.github.io/bfg-repo-cleaner/

OriolAbril commented:
Deleting the file plus then us doing squash and merge should not include the nc files anywhere in the fit tree, I think there is no need to an extra rebase

canyon289 commented:
> Deleting the file plus then us doing squash and merge should not include the nc files anywhere in the fit tree, I think there is no need to an extra rebase

Ah yea that's easier.

sudip-padhye committed:
Removed Plots and changed saving model method

sudip-padhye committed:
Removed Plots and changed saving model method

OriolAbril commented:
This is great @sudip-padhye ! I'll merge once you delete the .nc files

sudip-padhye committed:
deleting nc files
